src/patternfly/components/Alert/docs/code.md

```diff
@@ -6,26 +6,26 @@ Always add a modifier class. Do not use `.pf-c-alert` on its own.
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `role="alert"` | `.pf-c-alert__body` |  Communicates contents of the alert message to the user. This should only be used on dynamically created alerts, such as toast notifications or inline notifications that aren't present during page load (for example, an alert at the top of a form when there are errors in the form). |
-| `aria-label="Success Notification"` | `.pf-c-alert` |  Indicates the Success Notification. |
-| `aria-label="Danger Notification"` | `.pf-c-alert` |  Indicates the Danger Notification. |
-| `aria-label="Warning Notification"` | `.pf-c-alert` |  Indicates the Warning Notification. |
-| `aria-label="Information Notification"` | `.pf-c-alert` |  Indicates the Information Notification. |
-| `aria-hidden="true"` | `.pf-c-alert__icon` |  Hides icon for assistive technologies. ** Required **|
+| `aria-label="Success alert"` | `.pf-c-alert` |  Indicates the Success alert. |
+| `aria-label="Danger alert"` | `.pf-c-alert` |  Indicates the Danger alert. |
+| `aria-label="Warning alert"` | `.pf-c-alert` |  Indicates the Warning alert. |
+| `aria-label="Information alert"` | `.pf-c-alert` |  Indicates the Information alert. |
+| `aria-label="Close Success Alert: Success alert title"` | `.pf-c-button.pf-c-plain` |  In the case of a close button it is recommended you provide more descriptive text than just `close`. |
```

Comment:
```suggestion
| `aria-label="Close Success Alert: Success alert title"` | `.pf-c-button.pf-m-plain` |  In the case of a close button it is recommended you provide more descriptive text than just "close". |
```

not sure about the "close" quotes. seems more like text than code though.

Reply:
I agree with using quotation marks here, although I think the sentence as a whole is a little unclear. What about something along the lines of:

> Close button labels should uniquely identify the alert by including its type and title

I think that would clarify why "Close Success Alert: Success alert title" is the way it is. Also the "A" in "Success Alert" could probably be lowercase.

Reply:
+1 I've updated this. The goal is to provide additional text to clarify which alert will be closed. We don't necessarily recommend using the whole title. We want to avoid the entire title being read twice. Let me know what you think of the update.

Reply:
That makes sense, I like the newly-updated text. 👍